### ISSUE TITLE
💅 In Svelte files, unsafe fixes aren't applied in inline event handlers with `--write --unsafe`

### ISSUE THREAD

**Issue by dyc3** [biomejs/biome#10247]
### Environment information

<details>

```bash
CLI:
  Version:                      2.4.14
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_PATH:                    unset
  BIOME_LOG_PREFIX_NAME:             unset
  BIOME_LOG_LEVEL:                   unset
  BIOME_LOG_KIND:                    unset
  BIOME_CONFIG_PATH:                 unset
  BIOME_THREADS:                     unset
  BIOME_WATCHER_KIND:                unset
  BIOME_WATCHER_POLLING_INTERVAL:    unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v22.22.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         bun/1.3.3

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  false
  HTML full support enabled:    true

Workspace:
  Open Documents:               0

```

</details>

### Rule name

useBlockStatements but i think it could apply to all rules

### Playground link

https://github.com/dyc3/biome-repro-use-block-svelte

### Expected result

it should write the fixes

also, fyi: i had the clanker create the reproduction, but i found the bug myself.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
